spec/requests/families_spec.rb

```diff
@@ -33,6 +69,15 @@
       expect(response).to redirect_to(family_path(family.id))
       expect(request.flash[:notice]).to eql "Family was successfully created."
     end
+
+    it "renders :new if the create action does not succeed" do
+      post families_path, params: { family:  { guardian_first_name: nil } }
+
+      family = Family.last
+
+      expect(response).to render_template(:new)
+      expect(family).to be_nil
```

Comment:
I think the right assertion to make here is more along the lines of...

```ruby
it "does not persist a Family record" do
  expect do
    post families_path, params: { family: { guardian_first_name: nil } }
  end.not_to change(Family, :count)	
end
```

And perhaps a second test that it returns a 200 response?